_layouts/post.html

```diff
@@ -1,6 +1,10 @@
 ---
 layout: default
 ---
+{% if page.features contains 'latex' %}
+<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
```

Comment:
cdn.mathjax.org has been retired, look into the source (or dev console when loading that file).
Other options are listed in the official docs: https://docs.mathjax.org/en/latest/web/start.html#using-mathjax-from-a-content-delivery-network-cdn

I'd also prefer to have a [SRI string](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) included, although that implies pinning the version:
```suggestion
<script src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js" integrity="sha512-ehyCwiYKL3B7bx/X+X0rs1JKF9ANzwjOeOvB69YoPOZnpBhC9GRSu2ry9vAuv4Z460WI7Yt4iq0dEPPeCElnXA==" crossorigin="anonymous" type="text/javascript"></script>
```